gadgets/top_file/program.bpf.c

```diff
@@ -30,20 +31,17 @@ enum type {
 struct file_id {
 	__u64 inode;
 	__u32 dev;
-	gadget_pid pid;
-	gadget_tid tid;
+	// TODO: how to skip this field in the output?
+	__u64 pid_tgid;
```

Comment:
Add hidden annotation in gadget.yaml?

How was it done before?

Reply:
The hidden annotation only hides this from the columns output. I added support for ignoring "private"
 fields on the ebpf operators. @flyth what do you think?